windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -31,23 +31,36 @@ endif::openshift-origin[]
 [id="wmco-6-0-0"]
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in https://access.redhat.com/errata/RHBA-[2022:97930].
+
+[NOTE]
+====
+Windows Server 2019 is not supported in vSphere.
+====
+
+=== Bug fixes
+Previously, a Windows instance could not be unconfigured when the node's external IP was present without a PTR (pointer record). As a result, the node's external IP address without reverse lookup records was used to associate the Windows instance.
+With this release, Windows instance de-configuration succeeds when the node's external IP address is present without a PTR .
```

Comment:
Is this meant to be a new line? If so, needs an additional line above. If not, should remove the break entirely.